wiring/src/spark_wiring_ble.cpp

```diff
@@ -2089,6 +2089,7 @@ int BleLocalDevice::setAdvertisingInterval(uint16_t interval) const {
     hal_ble_adv_params_t advParams = {};
     advParams.size = sizeof(hal_ble_adv_params_t);
     CHECK(hal_ble_gap_get_advertising_parameters(&advParams, nullptr));
+    advParams.version = BLE_API_VERSION;
```

Comment:
The version should be set before getting the current params. Same to elsewhere the version is set.